docker-compose.yaml

```diff
@@ -16,11 +16,10 @@ services:
          condition: service_healthy
       rabbitmq:
          condition: service_healthy
-    #  - "ipfs"
     volumes:
       - local_cache:/home/ubuntu/local_cache/
     links:
-    #  - "ipfs"
+      - "ipfs"
```

Comment:
This won't work if an IPFS URL is passed since the profile won't be pulled, right?